examples/17-drawstress/drawstress.cpp

```diff
@@ -208,20 +208,13 @@ class ExampleDrawStress : public entry::AppI
 	{
 		for (;;)
 		{
-			union
-			{
-				void* ptr;
-				uintptr_t id;
-
-			} cast;
-
-			cast.ptr = _thread->pop();
-			if (UINTPTR_MAX == cast.id)
+			uintptr_t id = reinterpret_cast<uintptr_t>(_thread->pop());
```

Comment:
Still don't like reinterpret cast... This is not better than union thing.

Reply:
I should have probably marked the current this variable `const` in the commit. The difference we'd have is that we can mark `uintptr_t id` const, but not the same with `union {} cast`.